src/views/GitHub.vue

```diff
@@ -1,10 +1,40 @@
 <template>
-
+    <v-flex xs2>
+        <v-navigation-drawer permanent absolute>
```

Comment:
@divya-yadamreddi there is a component in the original code for this - `NavDrawerTemporary.vue`. Please use that.